src/linux/system.rs

```diff
@@ -507,6 +507,17 @@ impl SystemExt for System {
         get_system_info("NAME=")
     }
 
+    fn get_host_name(&self) -> Option<String> {
+        let hostname_max = unsafe { sysconf(_SC_HOST_NAME_MAX) };
+        let mut buffer = vec![0_u8; (hostname_max as usize) + 1];
```

Comment:
No need for an extra `\0` here. We will remove it anyway. ;)

Reply:
removed the `+1` :D